CHANGELOG.md

```diff
@@ -1,4 +1,38 @@
 ## 0.5.0 (Unreleased)
+
```

Comment:
Ah, .5 does make sense.  I was thinking this would be 0.4.3 but since we added a new resource 0.5.0 makes more sense.  

General fun note: We will bump minor versions on the way to 1.0.0 when we add new resources - with this release now we cover 17 out of approximately 24 Atlas resources supportable with Terraform (and 3 we hope to support when they are ready). So when we are at the 24 we'll hit version 1.

Reply:
Awesome we are soo close!